src/auction.js

```diff
@@ -691,7 +692,7 @@ export function getStandardBidderSettings(mediaType, bidderCode) {
     });
 
     // Adding hb_cache_host
-    if (config.getConfig('cache.url') && (!bidderCode || deepAccess(bidderSettings, `${bidderCode}.sendStandardTargeting`) !== false)) {
+    if (config.getConfig('cache.url') && (!bidderCode || bidderSettings.get(bidderCode, 'sendStandardTargeting') !== false)) {
```

Comment:
This is potentially a breaking change: previously, this would be evaluated the first time this function was called with `mediaType === 'video'` and would set `hb_cache_host` globally after that. There was also no fallback to `bidderSettings.standard`.

Reply:
Breaking change... more like a bug fix!